docs/products/colors.mdx

```diff
@@ -57,6 +57,9 @@ import ColorBlock from './components/colorblock.js'
   </div>
   <div className="col-sm-3">
     <ColorBlock name={dodgerblue} />
+  </div>
+    <div className="col-sm-3">
+    <ColorBlock name={terminalblue} />
```

Comment:
import this color at the beginning of the page

Reply:
- Added